Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -1415,5 +1415,10 @@ public List<string> SelectedTexts
     /// </summary>
     [Parameter] public bool AutoSelectFirstItem { get; set; }
 
+    /// <summary>
+    /// Gets or sets whether the autocomplete will use the Virtualize functionality.
+    /// </summary>
+    [Parameter] public bool Virtualize { get; set; }
+
```

Comment:
Like in DataGrid, make available a **VirtualizeOptions** parameter across all components that use **Virtualzie**?

Reply:
I guess we don't need this.